### PR TITLE
Fix/stick screed dep

### PR DESCRIPTION
**Pull request by mr-c**
This isn't going to be merged, just a review for the 1.4.1 release

### CODE REVIEW

**Comment by mr-c** [comment]
@ctb ready for review

**Comment by mr-c** [comment]
```
Removing screed 0.9 from easy-install.pth file
Adding screed 0.8 to easy-install.pth file
```

The version specifier works as intended

**Comment by mr-c** [comment]
Release checklist including BatLab is finished and all is ready.

File: doc/release-notes/release-1.4.1.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# khmer v1.4.1 release notes
+
+This is the v1.4.1 release of khmer. Due to the Python 3 migration we need to
```
Comment (ctb) [inline doc/release-notes/release-1.4.1.md]: suggest referring to issue # here (for Python 3 migration) OR changing it to be "Due to the upcoming Python 3 compatibility that will be part of khmer 2.0" or some more info.

Comment (ctb) [inline doc/release-notes/release-1.4.1.md]: (You explain it below quite well, I note. But it needs to be expanded on here otherwise will derail)

**Comment by ctb** [comment]
LGTM.

**Comment by mr-c** [comment]
@ctb wording tweaked; opinion?

**Comment by ctb** [comment]
Looks good!

**Pull request closed without merging**